### DISCUSSION TITLE
Premium In-depth Learning Resource

### DISCUSSION THREAD

**Discussion by ixxie** [helix-editor/helix#11934] [Ideas]
I've started using Helix as my primary editor, and I'm very grateful. Its a spectacular experience :100:  :boom:  :dna:  :purple_heart:  Today I became an Open Collective backer as a way of giving something back. :pray: 

But I feel like I'm just scratching the surface, and I would really like to get deeper expertise. Knowing time and energy is limited, I would happily **pay** for an **in-depth premium learning resource**.

This could take several forms:
- Simple files (`hx --tutor` style)
- An interactive program
- Web content

Even better, some mix of the above : my dream course would consist of an explanatory video for each theme + tutorial files + an interactive program providing feedback as I follow the tutorial files.

Its important this sort of thing goes deep, because the whole point is to gain some editing superpowers from the course.

Ideally it would be some one-off payment for lifetime access, but wouldn't mind a SaaS style fee.

**Comment by gabydd** [comment]
@tomgroenwoldt has done some interesting open source projects in this space such as: https://github.com/tomgroenwoldt/helix-shortcut-quiz and keyglide(similar to vim golf change a buffer to the correct output in the shortest keystrokes and shortest time) which is currently being rewritten

**Reply by tomgroenwoldt** [reply]
Hey, thanks for mentioning me here. I rebuilt a POC, now it's live here: https://keygli.de.

**Comment by David-Else** [comment]
I am working on it! https://helix-editor-tutorials.com/mastering-the-helix-editor-udemy-course/

**Reply by ixxie** [reply]
Signed up updates! Any idea when it will be ready?

**Reply by David-Else** [reply]
Thanks! Not yet, but you will be among the first to know having signed up for the newsletter.

**Comment by jgarte** [comment]
Ideally, we'd just improve `hx --tutor` and keep things in the commons for everyone.
